### PR TITLE
Export file encoding

### PR DESCRIPTION
**Pull request by junajan**
This adds users possibility to specify output encoding of exported file.
Base branch is used that one with fullExport feature so we won't have conflicts later when merging to master

### CODE REVIEW

**Comment by PhilippSpo** [comment]
tests? 😜

**Comment by junajan** [comment]
Ahh.. pardon @PhilippSpo .. this should be peace of cake

**Comment by junajan** [comment]
@PhilippSpo - here it is 03f44b0

File: src/coffee/export.coffee [hunk 1]
```diff
@@ -305,8 +305,7 @@ class Export
 
       if file
         parsedCsv.to.string (res) =>
-          if append then res = '\n'+res
-          converted = iconv.encode(res, @options.encoding)
+          converted = iconv.encode(res+'\n', @options.encoding)
```
Comment (PhilippSpo) [inline src/coffee/export.coffee]: Why do you need the \n in the first place?

File: src/spec/integration/export.spec.coffee [hunk 2]
```diff
@@ -205,3 +206,36 @@ describe 'Export integration tests', ->
...
     .finally ->
       tempDir.removeCallback()
+
+  it 'should export labels of lenum and set of lenum', (done) ->
+    encoding = 'win1250'
+    template =
+    '''
+      productType,name,variantId,attr-lenum-n.en,attr-set-lenum-n.en,žškřďťň
+      '''
+    file = '/tmp/output.csv'
```
Comment (PhilippSpo) [inline src/spec/integration/export.spec.coffee]: Can we name this `outputLocation`? Every time I read file, I expect to have a file, not a path to a file 😜

Comment (junajan) [inline src/spec/integration/export.spec.coffee]: sure .. I took it from previous tests so I will rename it everywhere

File: src/coffee/export.coffee (hunk 1 above)
Comment (junajan) [inline src/coffee/export.coffee]: This preserve backward compatibility - without this `\n` original tests did not work

File: src/spec/integration/export.spec.coffee (hunk 2 above)
Comment (junajan) [inline src/spec/integration/export.spec.coffee]: Updated here - b51bcd0

File: src/spec/integration/export.spec.coffee [hunk 3]
```diff
@@ -205,3 +206,36 @@ describe 'Export integration tests', ->
     .catch (err) -> done _.prettify(err)
     .finally ->
       tempDir.removeCallback()
+
+  it 'should export labels of lenum and set of lenum', (done) ->
```
Comment (hisabimbola) [inline src/spec/integration/export.spec.coffee]: I think this test is really about exporting labels with a different **encoding**. We can make the test description more descriptive that that is what this test case is trying to achieve and not just export labels.

Comment (junajan) [inline src/spec/integration/export.spec.coffee]: Ahh.. sorry - that's mistake of course

Comment (junajan) [inline src/spec/integration/export.spec.coffee]: fixed here 924cfd2

**Pull request merged**